src/components/RadioBorderedBox/index.tsx

```diff
@@ -104,54 +73,56 @@ const RadioBorderedBox = ({
   error,
 }: RadioBorderedBoxProps) => (
   <StyledMainContainer>
-    <StyledBorderedBox disabled={disabled} checked={checked} error={error}>
-      <StyledSpaceContainer>
-        <StyledRadioContainer>
-          <Radio
-            name={name}
-            checked={checked}
-            onChange={onChange}
-            onFocus={onFocus}
-            onBlur={onBlur}
-            disabled={disabled}
-            value={value}
-            error={error}
-          >
+    <SelectableCard
+      value={value}
+      onChange={onChange}
+      disabled={disabled}
+      name={name}
+      checked={checked}
+      isError={!!error}
+      onFocus={onFocus}
+      onBlur={onBlur}
+      type="radio"
+      showTick
+    >
+      <StyledMainContainer>
+        <StyledSpaceContainer>
+          <StyledRadioContainer>
             {label}
-          </Radio>
-          {labelDescription ? (
-            <Text as="span" variant="body" disabled={disabled}>
-              {labelDescription}
-            </Text>
-          ) : null}
-          {badgeText && (
-            <Badge
-              size={badgeSize}
-              variant={badgeVariant}
-              prominence={badgeProminence}
+            {labelDescription ? (
+              <Text as="span" variant="body" disabled={disabled}>
+                {labelDescription}
+              </Text>
+            ) : null}
+            {badgeText && (
```

Comment:
Lets use a ternary instead:

```suggestion
            {badgeText ? (
```